jsonrpc/filter_manager_test.go

```diff
@@ -11,6 +13,123 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func Test_GetLogsForQuery(t *testing.T) {
+	t.Parallel()
+
+	blockHash := types.StringToHash("1")
+
+	// Topics we're searching for
+	topic1 := types.StringToHash("4")
+	topic2 := types.StringToHash("5")
+	topic3 := types.StringToHash("6")
+
+	var topics = [][]types.Hash{{topic1}, {topic2}, {topic3}}
```

Comment:
This can be a helper method

Reply:
Not sure what should be here a helper method

Reply:
Helper method like `generateTopics`, where you pass in the number of topics you want